src/components/DocumentationTopic.vue

```diff
@@ -734,7 +734,8 @@ $space-size: 15px;
     top: 50%;
     left: 50%;
     transform: translate(-50%, -50%);
-    background-color: var(--color-fill); // cover border line
+    // cover border line
+    background-color: var(--colors-text-background, var(--color-other-decl-button));
```

Comment:
```suggestion
    background-color: var(--color-other-decl-button);
```

It's never not defined, so I don't think a fallback is needed.

Reply:
I don't believe so. Theme-settings can override _any_ color var, and the [spec][1] uses the singular var name that is defined where the app defines all the shared colors.

There is an IDE-specific feature for synchronizing the IDE code syntax colors in `src/components/Tutorial/CodeTheme.vue`, but it is only used for code listings and not generic app colors—it shouldn't apply here.

Either way, the more specific color should be tried first with the more generic one being a fallback. Since this new color is already derived from an existing one, users could still override either just the text background to also apply it here, or they could override both to have a specific color for each, although that would be unlikely since this only has a specific color meant to match the background and override the border here, which is a very specific use case.

[1]: https://github.com/apple/swift-docc/blob/e6b81527f9f46347ededad3464b1f3585c81369d/Sources/SwiftDocC/SwiftDocC.docc/Resources/ThemeSettings.spec.json#L78